components/tag-list/tag-list.js

```diff
@@ -313,10 +311,6 @@ class TagList extends LocalizeCoreElement(InteractiveMixin(ArrowKeysMixin(LitEle
 			if (node.nodeType !== Node.ELEMENT_NODE) return false;
 			await node.updateComplete;
 
-			const role = node.getAttribute('role');
-			const backdropRole = node.getAttribute(BACKDROP_ROLE);
-			if (role !== 'listitem' && backdropRole !== 'listitem') return false;
```

Comment:
Now that we're no longer checking the role, are we getting any additional nodes?  Or was this just being cautious to begin with?